SR2025/2024-09-16-place-rejection.md

```diff
@@ -0,0 +1,28 @@
+---
+to: Student Robotics 2025 rejections
+subject: Student Robotics 2025 is sadly full
+---
+
+Hi,
+
+We had a lot more interest in Student Robotics 2025 than we had places
+available. This has meant that unfortunately we're unable to offer you a place
+this year.
+
+We realise this is disappointing (we're disappointed too!), however we are
+limited by the number of kits we have and by the size of the venue. We are
+always looking for ways to support more teams and hope to be able to do so in
+future.
+
+We've put you on our waiting list so if we have any early drop outs we will be
+sure to let you know. You may be able to take their place, but we cannot promise
+anything. Let us know if you don't want to be on the waiting list.

```

Comment:
Do we want to say these to all teams that didn't make it? I don't know how many rejects there are, but I don't think it's worth having more than a few teams on the waiting list?

Reply:
I think this is what we've done in the past. We don't know if teams who are on the waiting list will actually want the place should one appear, so it's probably better to have a few more than we need.

If we have a deadline after which we won't reallocate a place I think that could also be informative around this. Let's discuss both aspects on Wednesday (I see the reallocation process is already on the agenda); I don't think this is urgent to send.

Reply:
We're not putting all teams on the waiting list, we are rejecting some teams. I think that means this is ok?

Reply:
Given we're doing two separate emails, yeah, let's keep this here and the other email can be different.